2015/02/12 01:32:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1044] Access denied for user 'revista'@'localhost' to database 'revista'' in C:\xampp\htdocs\topicos\yii\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\topicos\yii\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\topicos\yii\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\topicos\yii\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\topicos\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\topicos\yii\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\topicos\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\topicos\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\topicos\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\topicos\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\topicos\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\topicos\yii\revista\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/topicos/yii/revista/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/topicos/yii/revista/index.php?r=gii
---
2015/02/12 01:43:51 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`revista`.`articulo`, CONSTRAINT `fk_articulo_indice_articulo_1` FOREIGN KEY (`id_indice_articulo`) REFERENCES `indice_articulo` (`id_indice_articulo`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `articulo` (`nombre`, `resumen`, `abstrac`, `introduccion`, `metodologia`, `contenido`, `fecha_creacion`, `archivo_pdf`, `id_status`, `conclusiones`, `agradecimientos`, `referencias`, `id_indice_articulo`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12).
in C:\xampp\htdocs\topicos\yii\revista\protected\controllers\ArticuloController.php (73)
in C:\xampp\htdocs\topicos\yii\revista\index.php (13)
2015/02/12 01:43:51 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`revista`.`articulo`, CONSTRAINT `fk_articulo_indice_articulo_1` FOREIGN KEY (`id_indice_articulo`) REFERENCES `indice_articulo` (`id_indice_articulo`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `articulo` (`nombre`, `resumen`, `abstrac`, `introduccion`, `metodologia`, `contenido`, `fecha_creacion`, `archivo_pdf`, `id_status`, `conclusiones`, `agradecimientos`, `referencias`, `id_indice_articulo`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12)' in C:\xampp\htdocs\topicos\yii\framework\db\CDbCommand.php:358
Stack trace:
#0 C:\xampp\htdocs\topicos\yii\framework\db\ar\CActiveRecord.php(1080): CDbCommand->execute()
#1 C:\xampp\htdocs\topicos\yii\framework\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\topicos\yii\revista\protected\controllers\ArticuloController.php(73): CActiveRecord->save()
#3 C:\xampp\htdocs\topicos\yii\framework\web\actions\CInlineAction.php(49): ArticuloController->actionCreate()
#4 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\topicos\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\topicos\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\topicos\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\topicos\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/create')
#14 C:\xampp\htdocs\topicos\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\topicos\yii\revista\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/topicos/yii/revista/index.php?r=articulo/create
HTTP_REFERER=http://localhost/topicos/yii/revista/index.php?r=articulo/create
---
2015/02/14 00:11:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "revista".' in C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('revista')
#1 C:\xampp\htdocs\topicos\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\topicos\yii\revista\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/topicos/yii/revista/index.php?r=revista
---
2015/02/14 00:15:07 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`revista`.`articulo`, CONSTRAINT `fk_articulo_indice_articulo_1` FOREIGN KEY (`id_indice_articulo`) REFERENCES `indice_articulo` (`id_indice_articulo`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `articulo` (`nombre`, `resumen`, `abstrac`, `introduccion`, `metodologia`, `contenido`, `fecha_creacion`, `archivo_pdf`, `id_status`, `conclusiones`, `agradecimientos`, `referencias`, `id_indice_articulo`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12).
in C:\xampp\htdocs\topicos\yii\revista\protected\controllers\ArticuloController.php (73)
in C:\xampp\htdocs\topicos\yii\revista\index.php (13)
2015/02/14 00:15:07 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`revista`.`articulo`, CONSTRAINT `fk_articulo_indice_articulo_1` FOREIGN KEY (`id_indice_articulo`) REFERENCES `indice_articulo` (`id_indice_articulo`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `articulo` (`nombre`, `resumen`, `abstrac`, `introduccion`, `metodologia`, `contenido`, `fecha_creacion`, `archivo_pdf`, `id_status`, `conclusiones`, `agradecimientos`, `referencias`, `id_indice_articulo`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12)' in C:\xampp\htdocs\topicos\yii\framework\db\CDbCommand.php:358
Stack trace:
#0 C:\xampp\htdocs\topicos\yii\framework\db\ar\CActiveRecord.php(1080): CDbCommand->execute()
#1 C:\xampp\htdocs\topicos\yii\framework\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\topicos\yii\revista\protected\controllers\ArticuloController.php(73): CActiveRecord->save()
#3 C:\xampp\htdocs\topicos\yii\framework\web\actions\CInlineAction.php(49): ArticuloController->actionCreate()
#4 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\topicos\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\topicos\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\topicos\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\topicos\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/create')
#14 C:\xampp\htdocs\topicos\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\topicos\yii\revista\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/topicos/yii/revista/index.php?r=articulo/create
HTTP_REFERER=http://localhost/topicos/yii/revista/index.php?r=articulo/create
---
2015/02/14 00:19:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "agregar".' in C:\xampp\htdocs\topicos\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(270): CController->missingAction('agregar')
#1 C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php(282): CController->run('agregar')
#2 C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/agrega...')
#3 C:\xampp\htdocs\topicos\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicos\yii\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicos/yii/revista/index.php?r=articulo/agregar
---
2015/02/14 00:54:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "create".' in C:\xampp\htdocs\topicos\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicos\yii\framework\web\CController.php(270): CController->missingAction('create')
#1 C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php(282): CController->run('create')
#2 C:\xampp\htdocs\topicos\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/create')
#3 C:\xampp\htdocs\topicos\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicos\yii\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicos/yii/revista/index.php?r=articulo/create
HTTP_REFERER=http://localhost/topicos/yii/revista/index.php?r=articulo/view&id=4
---
2015/02/19 00:20:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "articulos/create".' in C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulos/creat...')
#1 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/topicosweb/11030704/revista/?r=articulos/create
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/
---
2015/02/19 00:22:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "articulos/create".' in C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulos/creat...')
#1 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/topicosweb/11030704/revista/?r=articulos/create
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/
---
2015/02/19 00:22:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "articulos/create".' in C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulos/creat...')
#1 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/topicosweb/11030704/revista/?r=articulos/create
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/
---
2015/02/19 00:22:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "articulos/create".' in C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulos/creat...')
#1 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/topicosweb/11030704/revista/?r=articulos/create
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/
---
2015/02/19 00:22:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "articulos/create".' in C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulos/creat...')
#1 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/topicosweb/11030704/revista/?r=articulos/create
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/
---
2015/02/19 00:23:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "create".' in C:\xampp\htdocs\topicosweb\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(270): CController->missingAction('create')
#1 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('create')
#2 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/create')
#3 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicosweb/11030704/revista/?r=articulo/create
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/
---
2015/02/19 00:23:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "create".' in C:\xampp\htdocs\topicosweb\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(270): CController->missingAction('create')
#1 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('create')
#2 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/create')
#3 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicosweb/11030704/revista/?r=articulo/create
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/
---
2015/02/19 00:31:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "agregar".' in C:\xampp\htdocs\topicosweb\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(270): CController->missingAction('agregar')
#1 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('agregar')
#2 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('revista/agregar')
#3 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicosweb/11030704/revista/?r=revista/agregar
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/?r=articulo/agregar
---
2015/02/19 00:31:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "crear".' in C:\xampp\htdocs\topicosweb\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(270): CController->missingAction('crear')
#1 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('crear')
#2 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('revista/crear')
#3 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicosweb/11030704/revista/?r=revista/crear
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/??r=autor/agregar
---
2015/02/19 00:32:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "crear".' in C:\xampp\htdocs\topicosweb\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(270): CController->missingAction('crear')
#1 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('crear')
#2 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('revista/crear')
#3 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicosweb/11030704/revista/?r=revista/crear
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/??r=autor/agregar
---
2015/02/19 00:37:16 [error] [php] Use of undefined constant form - assumed 'form' (C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php:24)
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#1 C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php(18): AutorController->renderPartial()
#2 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(95): AutorController->renderInternal()
#4 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(782): AutorController->renderPartial()
#6 C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php(79): AutorController->render()
#7 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): AutorController->actionCreate()
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(133): AutorController->runAction()
#10 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): AutorController->filterAccessControl()
#13 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): AutorController->runActionWithFilters()
#16 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): AutorController->run()
#17 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#19 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CWebApplication->run()
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=autor/create
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php (24)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php (18)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php (79)
2015/02/19 00:38:22 [error] [php] Use of undefined constant form - assumed 'form' (C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php:24)
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#1 C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php(18): AutorController->renderPartial()
#2 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(95): AutorController->renderInternal()
#4 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(782): AutorController->renderPartial()
#6 C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php(79): AutorController->render()
#7 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): AutorController->actionCreate()
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(133): AutorController->runAction()
#10 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): AutorController->filterAccessControl()
#13 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): AutorController->runActionWithFilters()
#16 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): AutorController->run()
#17 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#19 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CWebApplication->run()
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=autor/create
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php (24)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php (18)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php (79)
2015/02/19 00:38:32 [error] [php] Use of undefined constant form - assumed 'form' (C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php:24)
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#1 C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php(18): AutorController->renderPartial()
#2 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(95): AutorController->renderInternal()
#4 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(782): AutorController->renderPartial()
#6 C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php(79): AutorController->render()
#7 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): AutorController->actionCreate()
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(133): AutorController->runAction()
#10 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): AutorController->filterAccessControl()
#13 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): AutorController->runActionWithFilters()
#16 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): AutorController->run()
#17 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#19 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CWebApplication->run()
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=autor/create
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php (24)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php (18)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php (79)
2015/02/19 00:38:34 [error] [php] Use of undefined constant form - assumed 'form' (C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php:24)
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#1 C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php(18): AutorController->renderPartial()
#2 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(95): AutorController->renderInternal()
#4 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(782): AutorController->renderPartial()
#6 C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php(79): AutorController->render()
#7 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): AutorController->actionCreate()
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(133): AutorController->runAction()
#10 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): AutorController->filterAccessControl()
#13 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): AutorController->runActionWithFilters()
#16 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): AutorController->run()
#17 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#19 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CWebApplication->run()
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=autor/create
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php (24)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php (18)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php (79)
2015/02/19 00:38:45 [error] [php] Use of undefined constant form - assumed 'form' (C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php:24)
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#1 C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php(18): AutorController->renderPartial()
#2 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(95): AutorController->renderInternal()
#4 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(782): AutorController->renderPartial()
#6 C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php(79): AutorController->render()
#7 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): AutorController->actionCreate()
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(133): AutorController->runAction()
#10 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): AutorController->filterAccessControl()
#13 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): AutorController->runActionWithFilters()
#16 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): AutorController->run()
#17 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#19 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CWebApplication->run()
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=autor/create
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php (24)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php (18)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php (79)
2015/02/19 00:38:48 [error] [php] Use of undefined constant form - assumed 'form' (C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php:24)
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#1 C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php(18): AutorController->renderPartial()
#2 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(95): AutorController->renderInternal()
#4 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(782): AutorController->renderPartial()
#6 C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php(79): AutorController->render()
#7 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): AutorController->actionCreate()
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(133): AutorController->runAction()
#10 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): AutorController->filterAccessControl()
#13 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): AutorController->runActionWithFilters()
#16 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): AutorController->run()
#17 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#19 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CWebApplication->run()
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=autor/create
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php (24)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php (18)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php (79)
2015/02/19 00:40:03 [error] [php] Use of undefined constant form - assumed 'form' (C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php:36)
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#1 C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php(18): AutorController->renderPartial()
#2 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(95): AutorController->renderInternal()
#4 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(782): AutorController->renderPartial()
#6 C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php(79): AutorController->render()
#7 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): AutorController->actionCreate()
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(133): AutorController->runAction()
#10 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): AutorController->filterAccessControl()
#13 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): AutorController->runActionWithFilters()
#16 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): AutorController->run()
#17 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#19 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CWebApplication->run()
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=autor/create
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php (36)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php (18)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php (79)
2015/02/19 00:40:07 [error] [php] Use of undefined constant form - assumed 'form' (C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php:36)
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#1 C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php(18): AutorController->renderPartial()
#2 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\topicosweb\framework\web\CBaseController.php(95): AutorController->renderInternal()
#4 C:\xampp\htdocs\topicosweb\framework\web\CController.php(869): AutorController->renderFile()
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(782): AutorController->renderPartial()
#6 C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php(79): AutorController->render()
#7 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): AutorController->actionCreate()
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(133): AutorController->runAction()
#10 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): AutorController->filterAccessControl()
#13 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): AutorController->runActionWithFilters()
#16 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): AutorController->run()
#17 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#19 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CWebApplication->run()
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=autor/create
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\_form.php (36)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\views\autor\create.php (18)
in C:\xampp\htdocs\topicosweb\11030704\revista\protected\controllers\AutorController.php (79)
2015/02/19 01:22:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "agregar".' in C:\xampp\htdocs\topicosweb\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(270): CController->missingAction('agregar')
#1 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('agregar')
#2 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/agrega...')
#3 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=articulo/agregar
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/index.php?r=revista/create
---
2015/02/19 01:22:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "agregar".' in C:\xampp\htdocs\topicosweb\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(270): CController->missingAction('agregar')
#1 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('agregar')
#2 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/agrega...')
#3 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=articulo/agregar
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/index.php?r=articulo/agregar
---
2015/02/19 01:22:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "agregar".' in C:\xampp\htdocs\topicosweb\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(270): CController->missingAction('agregar')
#1 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('agregar')
#2 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/agrega...')
#3 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=articulo/agregar
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/index.php?r=revista/create
---
2015/02/19 01:24:11 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\topicosweb\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('agregar')
#8 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/agrega...')
#9 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=articulo/agregar
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/index.php?r=revista/create
---
2015/02/19 01:24:16 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\topicosweb\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('agregar')
#8 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/agrega...')
#9 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=articulo/agregar
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/index.php?r=articulo/agregar
---
2015/02/19 01:24:19 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\topicosweb\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('agregar')
#8 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/agrega...')
#9 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=articulo/agregar
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/index.php?r=revista/create
---
2015/02/19 01:24:46 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\topicosweb\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('agregar')
#8 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/agrega...')
#9 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=articulo/agregar
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/index.php?r=articulo/agregar
---
2015/02/19 01:25:08 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\topicosweb\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\topicosweb\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\topicosweb\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\topicosweb\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\topicosweb\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('agregar')
#8 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/agrega...')
#9 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\topicosweb\11030704\revista\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/topicosweb/11030704/revista/index.php?r=articulo/agregar
HTTP_REFERER=http://localhost/topicosweb/11030704/revista/index.php?r=articulo/agregar
---
2015/02/21 00:23:48 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'revista'@'localhost' (using password: YES)' in C:\xampp\htdocs\topicosweb\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\topicosweb\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\topicosweb\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\topicosweb\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\topicosweb\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\xampp\htdocs\topicosweb\framework\web\CFormModel.php(39): CrudCode->init()
#6 C:\xampp\htdocs\topicosweb\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\topicosweb\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\topicosweb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\topicosweb\11030802\revista\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/topicosweb/11030802/revista/index.php?r=gii/crud/index
HTTP_REFERER=http://localhost:8080/topicosweb/11030802/revista/index.php?r=gii/default/index
---
2015/02/21 00:44:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "create".' in C:\xampp\htdocs\topicosweb\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(270): CController->missingAction('create')
#1 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('create')
#2 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/create')
#3 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicosweb\11030802\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicosweb/11030802/revista/index.php?r=articulo/create
HTTP_REFERER=http://localhost:8080/topicosweb/11030802/revista/index.php?r=articulo/index
---
2015/02/21 00:55:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "create".' in C:\xampp\htdocs\topicosweb\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(270): CController->missingAction('create')
#1 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('create')
#2 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/create')
#3 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicosweb\11030802\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicosweb/11030802/revista/index.php?r=articulo/create
HTTP_REFERER=http://localhost:8080/topicosweb/11030802/revista/index.php?r=articulo/index
---
2015/03/04 16:49:57 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.nombre" is not defined.' in C:\xampp\htdocs\topicosweb\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\topicosweb\11030802\revista\protected\components\UserIdentity.php(19): CComponent->__get('nombre')
#1 C:\xampp\htdocs\topicosweb\11030802\revista\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\topicosweb\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\topicosweb\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\topicosweb\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 C:\xampp\htdocs\topicosweb\11030802\revista\protected\controllers\SiteController.php(95): CModel->validate()
#6 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\topicosweb\11030802\revista\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/topicosweb/11030802/revista/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/topicosweb/11030802/revista/index.php?r=site/login
---
2015/03/04 16:50:15 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.nombre" is not defined.' in C:\xampp\htdocs\topicosweb\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\topicosweb\11030802\revista\protected\components\UserIdentity.php(19): CComponent->__get('nombre')
#1 C:\xampp\htdocs\topicosweb\11030802\revista\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\topicosweb\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\topicosweb\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\topicosweb\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 C:\xampp\htdocs\topicosweb\11030802\revista\protected\controllers\SiteController.php(95): CModel->validate()
#6 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\topicosweb\11030802\revista\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/topicosweb/11030802/revista/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/topicosweb/11030802/revista/index.php?r=site/login
---
2015/03/04 16:54:16 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.nombre" is not defined.' in C:\xampp\htdocs\topicosweb\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\topicosweb\11030802\revista\protected\components\UserIdentity.php(19): CComponent->__get('nombre')
#1 C:\xampp\htdocs\topicosweb\11030802\revista\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\topicosweb\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\topicosweb\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\topicosweb\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 C:\xampp\htdocs\topicosweb\11030802\revista\protected\controllers\SiteController.php(95): CModel->validate()
#6 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\topicosweb\11030802\revista\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/topicosweb/11030802/revista/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/topicosweb/11030802/revista/index.php?r=site/login
---
2015/03/04 16:54:35 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.nombre" is not defined.' in C:\xampp\htdocs\topicosweb\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\topicosweb\11030802\revista\protected\components\UserIdentity.php(19): CComponent->__get('nombre')
#1 C:\xampp\htdocs\topicosweb\11030802\revista\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\topicosweb\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\topicosweb\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\topicosweb\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 C:\xampp\htdocs\topicosweb\11030802\revista\protected\controllers\SiteController.php(95): CModel->validate()
#6 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\topicosweb\11030802\revista\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/topicosweb/11030802/revista/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/topicosweb/11030802/revista/index.php?r=site/login
---
2015/03/04 16:54:53 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.nombre" is not defined.' in C:\xampp\htdocs\topicosweb\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\topicosweb\11030802\revista\protected\components\UserIdentity.php(19): CComponent->__get('nombre')
#1 C:\xampp\htdocs\topicosweb\11030802\revista\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\topicosweb\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\topicosweb\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\topicosweb\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 C:\xampp\htdocs\topicosweb\11030802\revista\protected\controllers\SiteController.php(95): CModel->validate()
#6 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\topicosweb\11030802\revista\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/topicosweb/11030802/revista/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/topicosweb/11030802/revista/index.php?r=site/login
---
2015/03/04 17:01:09 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.nombre" is not defined.' in C:\xampp\htdocs\topicosweb\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\topicosweb\11030802\revista\protected\components\UserIdentity.php(19): CComponent->__get('nombre')
#1 C:\xampp\htdocs\topicosweb\11030802\revista\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\topicosweb\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\topicosweb\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\topicosweb\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 C:\xampp\htdocs\topicosweb\11030802\revista\protected\controllers\SiteController.php(95): CModel->validate()
#6 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\topicosweb\11030802\revista\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/topicosweb/11030802/revista/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/topicosweb/11030802/revista/index.php?r=site/login
---
2015/03/04 17:11:13 [error] [exception.CException] exception 'CException' with message 'Property "UserIdentity.nombre" is not defined.' in C:\xampp\htdocs\topicosweb\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\topicosweb\11030802\revista\protected\components\UserIdentity.php(19): CComponent->__get('nombre')
#1 C:\xampp\htdocs\topicosweb\11030802\revista\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\topicosweb\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#3 C:\xampp\htdocs\topicosweb\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 C:\xampp\htdocs\topicosweb\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#5 C:\xampp\htdocs\topicosweb\11030802\revista\protected\controllers\SiteController.php(95): CModel->validate()
#6 C:\xampp\htdocs\topicosweb\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\topicosweb\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\topicosweb\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\topicosweb\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\topicosweb\11030802\revista\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/topicosweb/11030802/revista/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/topicosweb/11030802/revista/index.php?r=site/login
---
2015/03/05 01:02:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "create".' in C:\xampp\htdocs\topicosweb\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\topicosweb\framework\web\CController.php(270): CController->missingAction('create')
#1 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(282): CController->run('create')
#2 C:\xampp\htdocs\topicosweb\framework\web\CWebApplication.php(141): CWebApplication->runController('articulo/create')
#3 C:\xampp\htdocs\topicosweb\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\topicosweb\11030906\revista\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/topicosweb/11030906/revista/index.php?r=articulo/create
HTTP_REFERER=http://localhost/topicosweb/11030906/revista/index.php?r=/site/index
---
